documentation/sdk/docs/how_tos/simple_transaction/request_funds.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: Request Funds
+description: 'How to programmatically request testnet funds.'
+image: /img/logo/iota_mark_light.png
+keywords:
+- how to
+- faucet
+- testnet
+- test funds
+- nodejs
+- python
+- rust
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import RustCode from "../../libraries/rust/how_tos/simple_transaction/request_funds/_code_sample.mdx";
+import NodejsCode from "../../libraries/nodejs/how_tos/simple_transaction/request_funds/_code_sample.mdx";
+import PythonCode from "../../libraries/python/how_tos/simple_transaction/request_funds/_code_sample.mdx";
+import RustOutput from "../../libraries/rust/how_tos/simple_transaction/request_funds/_expected_output.mdx";
+import NodejsOutput from "../../libraries/nodejs/how_tos/simple_transaction/request_funds/_expected_output.mdx";
+import PythonOutput from "../../libraries/python/how_tos/simple_transaction/request_funds/_expected_output.mdx";
+import TipFaucet from "../../_admonitions/_tip_faucet.md";
+
+Once you have created your account<!-- TODO [created your account](01_create_a_wallet_account.mdx) -->,
+you can start using wallet.rs for value transactions. Since these examples target the testnet, you can request funds
+using the `request_funds_from_faucet` function.
+
+<TipFaucet/>
+
+## Code Example
+
+The following example will:
+
+1. Create an account manager.
```

Comment:
There is no account manager anymore, it's called wallet now

Reply:
```suggestion
1. Create a wallet.
```